cla_backend/apps/call_centre/tests/test_forms.py

```diff
@@ -453,12 +453,10 @@ def _test(case, datetime, error_msg):
         # invalid format
         _test(case, now_utc.strftime("%Y/%m/%d %H:%M"), u"Enter a valid date/time.")
 
-        # datetime in the past
-        _test(case, self._strftime(now_utc), u"Specify a date not in the current half hour.")
-
```

Comment:
Isn't this test fine? Can we not change the comment to `datetime in the current callback window` or `datetime which is too early`

Reply:
No, that would fail. That "comment" is the expected error message when running `_test()`. With the new rules, we would not expect an error when using the current datetime.